outputs.tf

```diff
@@ -229,3 +229,22 @@ output "ephemeral_block_device" {
     null
   )
 }
+
+################################################################################
+# Elastic IP
+################################################################################
+
+output "eip_public_id" {
```

Comment:
all of these variables and outputs should be stripped of `public` - there is no notion of a private elastic IP, its just an elastic IP (eip)